Mathlib/CategoryTheory/Limits/Shapes/Products.lean

```diff
@@ -49,9 +49,10 @@ variable {β : Type w}
 
 variable {C : Type u} [Category.{v} C]
 
--- We don't need an analogue of `pair` (for binary products), `parallel_pair` (for equalizers),
--- or `(co)span`, since we already have `discrete.functor`.
-attribute [local tidy] tactic.discrete_cases
+-- porting note: ...
+-- We don't need an analogue of `Pair` (for binary products), `ParallelPair` (for equalizers),
+-- or `(Co)span`, since we already have `Discrete.functor`.
+--attribute [local tidy] tactic.discrete_cases
```

Comment:
```suggestion
-- We don't need an analogue of `Pair` (for binary products), `ParallelPair` (for equalizers),
-- or `(Co)span`, since we already have `Discrete.functor`.
--attribute [local tidy] tactic.discrete_cases -- Porting note: no tidy
```